server/spinwick.go

```diff
@@ -220,11 +336,18 @@ func (s *Server) createSpinWick(pr *model.PullRequest, size string, withLicense
 
 func (s *Server) handleUpdateSpinWick(pr *model.PullRequest, withLicense bool) {
 	// other repos we are not updating
-	if pr.RepoName != "mattermost-server" && pr.RepoName != "mattermost-webapp" {
-		return
+	request := &spinwick.Request{
+		InstallationID: "n/a",
+		Error:          nil,
+		ReportError:    false,
+		Aborted:        false,
+	}
+	if pr.RepoName == "mattermost-server" || pr.RepoName == "mattermost-webapp" {
```

Comment:
we can have other repos that spin MM servers not only "mattermost-webapp" or "mattermost-server"

maybe just check if is from "customer-web-server" then call updateKubeSpinWick otherwise go straight to updateSpinWick